app/routes/events/components/JoinEventForm.js

```diff
@@ -202,6 +210,13 @@ class JoinEventForm extends Component<Props> {
                 ? 'Åpnet '
                 : 'Åpner '}
               <Time time={event.activationTime} format="nowToTimeInWords" />
+              {this.sumPenalties() > 0 && (
```

Comment:
Remember that penalties will only affect the registration time (or being added to the waiting list) if `event.heedPenalties` is set to `true`. Aka. if the event doesn't give penalties for not attending/unregistering too late, penalties will not matter. This is the case for events arranged by externals, most commonly IDI.

Reply:
I think line 207 `{!formOpen && ...` solves this?